sscanss/editor/editor.py

```diff
@@ -15,10 +15,19 @@ def __init__(self, parent):
 
         super().__init__(parent)
 
+        self.updateStyle(parent.editor_font_family, parent.editor_font_size)
+
+    def updateStyle(self, font_family, font_size):
+        """Updates the editor style. 
```

Comment:
I think "Updates the editor font" is a clearer docstring and updateFont is a clearer method name. There are many thing that could be associated with style. Also do you need to pass `font_family` and `font_size` into the method? it might be cleaner to get these from the parent i.e. `font.setFamily(self.parent.editor_font_family)` then the method can be called with no arguments